src/content/docs/errors-inbox/apm-tab.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: 'APM: Group errors tab '
+tags:
+  - APM
+  - APM UI pages
+  - Error analytics
+metaDescription: New Relic's error profiles feature in the classic APM view shows trends and anomalies that help you compare and troubleshoot error events.
+redirects:
+  - /docs/apm/applications-menu/error-analytics/error-profiles-troubleshoot-trends
+  - /docs/apm/applications-menu/error-analytics/apm-error-profiles-troubleshoot-trends
+---
+
+import apmGroupErrorsTab from 'images/apm_screenshot-full_group-errors-tab.png'
+
+import apmErrorProfiles from 'images/apm_screenshot-full_error-profiles_2.png'
+
+import apmTriageErrors from 'images/apm_screenshot-full_triage-errors.png'
+
+import apmErrorsOutsideTransactions from 'images/apm_screenshot-full_errors-outside-transactions.png'
+
+The APM group errors tab allows engineering teams to monitor errors in their applications and delve deeply into any issues. 
+
+Let's say you're a lead engineer for a messaging application and you get a ping from one of your stakeholders that says that users are reporting issues with the **Send Message** button. That's a fundamental problem with your application's performance and you'll need to understand what's going wrong and how to fix it as quickly as possible. The first thing you'll do is open the **Group errors** tab in the specific application that's experiencing issues. 
+
+<img
+  title="group errors tab"
+  alt="A depiction of the group errors tab"
+  src={apmGroupErrorsTab}
+/>
+
+## Errors list view  [#list-view]
+
+As you try to investigate why the **Send Message** button isn't working in your application, the first place to start is the **Errors list view**. Here you can see error rate charts at a glance and determine if there are any unexpected spikes, dips, or patterns with errors in general.
+
+The **Errors list view** allows you to:
+
+* Correlate any general patterns on the top 5 errors chart to alerts occurring during the same time period.
+* Dynamic grouping: The default grouping for error occurrences is based on error message, error class, and Transaction UI name. You have the option to change the grouping options by any attribute, up to five attributes at a time.
+* Filtering: Many New Relic customers instrument custom attributes and filtering on a specific custom attribute can be a really quick way to cut through the noise of all error occurrences. 
+
+## Errors details view [#details-view]
+
+Let's say that after you investigated your errors as a whole using the list view, you identified a specific group of hosts that is causing an error spike. You can now use the **details view** to better understand in-context details.
```

Comment:
```suggestion
Let's say that after you investigated your errors as a whole using the list view, you identified a specific group of hosts that is causing an error spike. You can now use the **Details view** to better understand in-context details.
```

Reply:
we either go lower case & non-bold if we're wording it like 'use the details view' or we go bold and sentence case if we're referencing UI wording, like 'use the **Details view**'. Both are valid.